catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -99,12 +125,40 @@ public void refreshRegistryEntries() throws FederationAdminException {
                 remoteMetacardsToCreate.add(remoteEntry.getValue());
             }
         }
+
         if (CollectionUtils.isNotEmpty(remoteMetacardsToUpdate)) {
             writeRemoteUpdates(remoteMetacardsToUpdate);
         }
         if (CollectionUtils.isNotEmpty(remoteMetacardsToCreate)) {
             createRemoteEntries(remoteMetacardsToCreate);
         }
+        if (enableDelete && CollectionUtils.isNotEmpty(remoteMetacardsToDelete)) {
```

Comment:
Could use `!remoteMetacardsToDelete.isEmpty()` since we know the collection will never be `null`.